recipes/httpd-configure.rb

```diff
@@ -41,3 +41,12 @@
 	supports :restart => true, :reload => true, :status => true
 	action [:enable, :start, :reload]
 end
+
+# Reload Apache periodically if we have memory leaks.
+# Not an ideal solution, but a universal one.
+file '/etc/cron.hourly/httpd-reload-on-low-memory' do
+	content "free -m |head -3 |tail -1 |awk '{print $4}' |grep '^[5-9][0-9][0-9]' > /dev/null || (service httpd status > /dev/null && service httpd reload)"
```

Comment:
can't this be a file that is then called by cron so we can have some comments etc on what this command is doing  etc.

Reply:
Can do.